examples/pysbd_as_spacy_component.py

```diff
@@ -3,27 +3,28 @@
 
 Installation:
 pip install spacy
+
+NOTE: Works with spacy>=3.x.x
 """
-import pysbd
 import spacy
+from spacy.language import Language
+
+from pysbd.utils import PySBDFactory
+
+
+@Language.factory("pysbd", default_config={"language": 'en'})
+def pysbd_component(nlp, name, language: str):
+    return PySBDFactory(nlp, language=language)
 
-def pysbd_sentence_boundaries(doc):
-    seg = pysbd.Segmenter(language="en", clean=False, char_span=True)
-    sents_char_spans = seg.segment(doc.text)
-    char_spans = [doc.char_span(sent_span.start, sent_span.end) for sent_span in sents_char_spans]
-    start_token_ids = [span[0].idx for span in char_spans if span is not None]
-    for token in doc:
-        token.is_sent_start = True if token.idx in start_token_ids else False
-    return doc
 
 if __name__ == "__main__":
     text = "My name is Jonas E. Smith.          Please turn to p. 55."
     nlp = spacy.blank('en')
 
     # add as a spacy pipeline component
-    nlp.add_pipe(pysbd_sentence_boundaries)
+    nlp.add_pipe("pysbd", first=True)
 
     doc = nlp(text)
     print('sent_id', 'sentence', sep='\t|\t')
     for sent_id, sent in enumerate(doc.sents, start=1):
-        print(sent_id, sent.text, sep='\t|\t')
+        print(sent_id, repr(sent.text), sep='\t|\t')
```

Comment:
Out of curiosity: why is the `repr()` necessary here?

Reply:
It's not necessary at all. I add it sometimes just to see raw representation of the sentence. Especially in given example to see whether trailing spaces are captured properly or not 😅 

Screenshot with `repr` vs without
<img width="467" alt="image" src="https://user-images.githubusercontent.com/15062408/176615096-cc28c7cd-0c4c-454d-84b7-5d59228ef495.png">